src/checkbox.jsx

```diff
@@ -37,6 +37,7 @@ const Checkbox = React.createClass({
     labelStyle: React.PropTypes.object,
     onCheck: React.PropTypes.func,
     unCheckedIcon: React.PropTypes.element,
+    valueLink: React.PropTypes.object,
```

Comment:
We should make this property deprecated since it's going to be the case in the next release of react.
We can use something like https://github.com/react-bootstrap/react-prop-types#deprecatedproptype-explanation